test/sass/mixins/typography.spec.scss

```diff
@@ -0,0 +1,41 @@
+@import "../test";
+@import "../../../source/css/sass/mixins/typography";
+
+@include describe("@mixin typography") {
+
+  @include it("generates superscript in font-variant-position") {
+
+    @include assert() {
+
+      @include output {
+        @include font-variant-position(super);
+      }
+
+      @include contains {
```

Comment:
I'd prefer `@include expect` (so the missing declarations would need to be included).